### PR TITLE
Database week1

### PR DESCRIPTION
**Pull request by Kositthai**
Submit Database lesson1

### CODE REVIEW

**Comment by dboyadzhiev** [comment]
Hi, 
could you pls split your PR in two: one for data bases and one for the JS :)

**Comment by Kositthai** [comment]
> 

I think I did it
Pls check again

File: databases/week1/the_task.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+SELECT * FROM `task` ORDER BY `created`;
+
+-- Get the single most recently created task --
+SELECT * FROM `task` ORDER BY `created` DESC LIMIT 1; 
+
+-- Get the title and due date of all tasks where the title or description contains database --
+SELECT title, due_date FROM `task` WHERE (title LIKE '%database%') OR (description LIKE '%database%');
+
+-- Get the title and status (as text) of all tasks --
+SELECT * FROM `task` JOIN `status` ON task.status_id = status.id;
```
Comment (dboyadzhiev) [inline databases/week1/the_task.sql:26]: I would pick only the fields I need
```suggestion
SELECT task.title, status.name FROM `task` JOIN `status` ON task.status_id = status.id; 
```

File: databases/week1/the_task.sql [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+SELECT * FROM `task` ORDER BY `created` DESC LIMIT 1; 
+
+-- Get the title and due date of all tasks where the title or description contains database --
+SELECT title, due_date FROM `task` WHERE (title LIKE '%database%') OR (description LIKE '%database%');
+
+-- Get the title and status (as text) of all tasks --
+SELECT * FROM `task` JOIN `status` ON task.status_id = status.id; 
+
+-- Get the name of each status, along with a count of how many tasks have that status --
+SELECT status.name AS 'Status', count(*) FROM `task` JOIN `status` ON task.status_id = status.id GROUP BY task.status_id;
```
Comment (dboyadzhiev) [inline databases/week1/the_task.sql:29]: In this case, the result is correct but more propper way is to use `status.name` in the `group by` and `task.id` in the `count` 
```suggestion
SELECT status.name AS 'Status', count(task.id) FROM `task` JOIN `status` ON task.status_id = status.id GROUP BY status.name; 
```

**Review by dboyadzhiev — CHANGES_REQUESTED** [review]

File: databases/week1/the_task.sql (hunk 2 above)
Comment (Kositthai) [reply databases/week1/the_task.sql]: > In this case, the result is correct but more propper way is to use `status.name` in the `group by` and `task.id` in the `count`

Thank you so much for your review!
I have edited my code as your suggestions, please let me now if I have to adjust anything else or I can merge my PR

**Review by dboyadzhiev — APPROVED** [review]

**Pull request merged**